### ISSUE TITLE
Make AudioStreamInterface local by default

### ISSUE THREAD

**Issue by epicEaston197** [Yellow-Dog-Man/Resonite-Issues#1867]
### Describe the bug?

For the same reasons in issue https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/1835#issuecomment-2078130932 for accessibility and safety reasons this should be considered

### To Reproduce

Spawn the AudioStreamInterface

### Expected behavior

It should be local by default

### Screenshots

N/A

### Reporters

#epicEaston197
